core/embed/trezorhal/stm32u5/secret.c

```diff
@@ -170,6 +170,29 @@ secbool secret_optiga_present(void) {
   return secret_present(SECRET_OPTIGA_KEY_OFFSET, SECRET_OPTIGA_KEY_LEN);
 }
 
+secbool secret_optiga_writable(void) {
+  uint32_t offset = SECRET_OPTIGA_KEY_OFFSET;
```

Comment:
`offset`, `len` can be `const`.
`secret` can be `const uint8_t *`.

Reply:
`const`-correctness might lead to smaller binary so might be worth doing across the codebase